### PR TITLE
Create ui-c8y-1018-503-105-adding-no-conflict-to-avoid-coliding-leaflet-versions-6622-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59809](https://cumulocity.atlassian.net/browse/MTM-59809)
Original PR: [6650](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6650)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-105-adding-no-conflict-to-avoid-coliding-leaflet-versions-6622-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: adding noConflict to avoid coliding leaflet versions (#6622) [GRAFT][release/y2024] (#6650)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-105-adding-no-conflict-to-avoid-coliding-leaflet-versions-6622-graft-release.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Avoid colliding Leaflet versions by adding noConflict
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-105-adding-no-conflict-to-avoid-coliding-leaflet-versions-6622-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59809
+version: 1018.503.105
+---
+adding noConflict to avoid coliding leaflet versions (#6622) [GRAFT][release/y2024] (#6650)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-105-adding-no-conflict-to-avoid-coliding-leaflet-versions-6622-graft-release.md:17]: The Claude AI suggested to use the following body:
```suggestion
In some scenarios, different versions of the Leaflet library used by Cumulocity IoT and custom widgets could collide and lead to unexpected behavior or errors. To address this, the noConflict mode has been added to Cumulocity IoT's Leaflet integration. With this change, custom widgets can now use their own Leaflet version without interfering with Cumulocity IoT's Leaflet version. This improves the reliability and compatibility when using Leaflet in custom widgets alongside Cumulocity IoT's built-in geospatial features.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1018-503-105-adding-no-conflict-to-avoid-coliding-leaflet-versions-6622-graft-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59809
+version: 1018.503.105
+---
+In some scenarios, different versions of the Leaflet library used by Cumulocity IoT and custom widgets could collide and lead to unexpected behavior or errors. To address this, the noConflict mode has been added to Cumulocity IoT's Leaflet integration. With this change, custom widgets can now use their own Leaflet version without interfering with Cumulocity IoT's Leaflet version. This improves the reliability and compatibility when using Leaflet in custom widgets alongside Cumulocity IoT's built-in geospatial features.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-105-adding-no-conflict-to-avoid-coliding-leaflet-versions-6622-graft-release.md:17]:
```suggestion
In some scenarios, different versions of the Leaflet library used by {{< product-c8y-iot >}} and custom widgets could collide and lead to unexpected behavior or errors. To address this, the noConflict mode has been added to {{< product-c8y-iot >}}'s Leaflet integration. With this change, custom widgets can now use their own Leaflet version without interfering with {{< product-c8y-iot >}}'s Leaflet version. This improves the reliability and compatibility when using Leaflet in custom widgets alongside {{< product-c8y-iot >}}'s built-in geospatial features.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**